frontend/components/MainPageV1/Home/Overview/Staking.tsx

```diff
@@ -77,7 +76,7 @@ const Streak = () => {
  */
 
 const DANGER_HOURS = 3;
-const SUCCESS_HOURS = 12;
+const WARNING_HOURS = 12;
```

Comment:
sorry hope it's fine to rename, figured that naming doesn't correspond to the purpose